tests/integration/security/sds_ingress/quic/ingress_test.go

```diff
@@ -78,6 +78,14 @@ func TestTlsGatewaysWithQUIC(t *testing.T) {
 		NewTest(t).
 		RequiresSingleCluster().
 		Run(func(t framework.TestContext) {
+			// Some cloud platform may throw the following error during creation of the service with mixed TCP/UDP protocols:
+			// "Error syncing load balancer: failed to ensure load balancer: mixed protocol is not supported for LoadBalancer".
+			// Make sure the service is up and running before proceeding with the test.
```

Comment:
nit: Probably we can remove these comments here (and below) as they are captured as part of the API definition.

Reply:
Removed